src/AuthorizationClient.js

```diff
@@ -800,11 +800,32 @@ class AuthorizationClient {
   }
 
   async Sign(message) {
-    return await Ethers.utils.joinSignature(
-      this.client.signer.signDigest ?
-        await this.client.signer.signDigest(message) :
-        await this.client.signer.signingKey.signDigest(message)
-    );
+    const signer = this.client.signer;
```

Comment:
if we are sure this only happens due to SetSignerFromWeb3Provider, can revert this change